helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/PerInstanceAccessor.java

```diff
@@ -287,6 +288,7 @@ public Response getInstanceConfig(@PathParam("clusterId") String clusterId,
   @Path("configs")
   public Response updateInstanceConfig(@PathParam("clusterId") String clusterId,
       @PathParam("instanceName") String instanceName, @QueryParam("command") String commandStr,
+      @DefaultValue("false") @QueryParam("doSanityCheck") boolean doSanityCheck,
```

Comment:
My suggestion is to set doSanityCheck's default value being true, i.e, we should always validate the instance config unless it is explicitly override to not doing so.

Reply:
Updated.